templates/admin/navbar.tmpl

```diff
@@ -35,9 +35,14 @@
 			</a>
 		{{end}}
 		{{if .EnableActions}}
-		<a class="{{if .PageIsAdminRunners}}active {{end}}item" href="{{AppSubUrl}}/admin/runners">
-			{{.locale.Tr "actions.runners"}}
-		</a>
+		<div class="item">
+			{{.locale.Tr "actions.actions"}}
+			<div class="menu">
```

Comment:
Is it (menu > item > menu) the best practice by Fomantic UI ?

Reply:
I copied this structure from the [fomantic sub menu demo](https://fomantic-ui.com/collections/menu.html#sub-menu) , semantic needs this structure to add `active`:

<img width="373" alt="Screen Shot 2023-04-25 at 12 57 33" src="https://user-images.githubusercontent.com/17645053/234178209-8275a279-5db6-4a8c-ba61-bb7eceeb2cee.png">


another style is dropdown submenu

<img width="457" alt="Screen Shot 2023-04-25 at 12 55 11" src="https://user-images.githubusercontent.com/17645053/234177983-0c021267-3915-4063-946a-5c2863c61ea6.png">